### PR TITLE
Methods/Summary updated, GTB complete for all missions

### LINKED ISSUE #69: Write up methods and results

### PR DESCRIPTION
**Pull request by steeleb**
Three small tasks with this:

1) This has a few small tweaks from copy/paste error. 

2) Updated .csv files to track performance metrics of the GTB models.

3) Updated Methods Summary with Markdown that closes #69.

Review should probably concentrate on the Methods/Results summary. I'll send the rendered .html in our email chain!

Tagging @matthewross07 just so he knows this has been updated!

### CODE REVIEW

File: Methods_Results_Summary.Rmd [hunk 1]
```diff
@@ -46,47 +73,92 @@ tm_basemap("CartoDB.Positron") +
...
+
 # Methods
 
 ## eePlumB
 
-Using the overarching architecture presented in the Global Rivers Obstruction Database (GROD) [@yang2022] to engage volunteer observers, we croudsourced class labels for Landsat and Sentinel-2 images for the following classes: 'cloud', 'open water', 'light near shore sediment', 'dark near shore sediment', 'offshore sediment', 'shoreline contamination', 'other', and 'algae bloom' using our Earth Engine Plume and Bloom labeling interface ("eePlumB"). Dates for labeling were limited to the months of April through November to avoid ice-on.
-
-In order to eliminate outlier band information and reduce noise in the input for our models, the second and ninety-eighth percentiles were calculated for each mission-band combination and label data associated with values outside of those cutoffs were dropped from the analysis. [[Could add the `02_label_class_summaries.Rmd` as supplemental.]]
+Using the overarching architecture presented in the Global Rivers Obstruction
+Database (GROD) [@yang2022] to engage volunteer observers, we croudsourced
```
Comment (lindsayplatt) [inline Methods_Results_Summary.Rmd:84]:
```suggestion
Database (GROD) [@yang2022] to engage volunteer observers, we crowdsourced
```

File: Methods_Results_Summary.Rmd [hunk 2]
```diff
@@ -122,7 +194,12 @@ filtered_label_table <- filtered_labels %>%
 label_table_join <- full_join(label_table, filtered_label_table)
 ```
 
-The collated crowdsourced label dataset consisted of `r nrow(labels)` labels for across all classes. There were `r nrow(ml_labels)` labels that were part of the classes of interest (cloud, open water, sediment). After filtering for outliers from each subset of mission-specific labels, there were `r nrow(filtered_labels)` labels with complete band information. Table 1 presents a break down of the labels.
+The collated crowdsourced label dataset consisted of `r nrow(labels)` labels
+for across all classes. There were `r nrow(ml_labels)` labels that were part of
```
Comment (lindsayplatt) [inline Methods_Results_Summary.Rmd:198]:
```suggestion
across all classes. There were `r nrow(ml_labels)` labels that were part of
```

File: Methods_Results_Summary.Rmd [hunk 3]
```diff
@@ -46,47 +73,92 @@ tm_basemap("CartoDB.Positron") +
 
 ```
 
+:"Area of interest for this analysis in purple, consisting of a portion the
+western extent of Lake Superior, the Apostle Islands, and Chequamegon Bay."
+
 # Methods
 
 ## eePlumB
```
Comment (lindsayplatt) [inline Methods_Results_Summary.Rmd:81]: It would be great to get some screenshots in here!

**Review by lindsayplatt — APPROVED** [review]
Hey B - left a few small comments. I think you are currently translating some of this content into Overleaf, so I think we could probably spend more time discussing figures/images over in that space.

File: Methods_Results_Summary.Rmd (hunk 3 above)
Comment (steeleb) [reply Methods_Results_Summary.Rmd:81]: totes!! I'm going to email about the time I have available for the next few weeks - we've got a deadline coming up that I need to focus on so deep work time is going to be limited on this project until after the 14th.

**Comment by steeleb** [comment]
Whoop!! Merging, and adding a few more issues based on our conversations in email!

**Pull request merged**